[DOC] Optimize the documentation to make it clearer and fix spelling errors (#4172)
strimzi/strimzi-kafka-operator: documentation/modules/configuring/ref-list-of-kafka-cluster-resources.adoc

`_cluster-name_-kafka-_idx_`:: Pods created by the Kafka StatefulSet.
`_cluster-name_-kafka-brokers`:: Service needed to have DNS resolve the Kafka broker pods IP addresses directly.
`_cluster-name_-kafka-bootstrap`:: Service can be used as bootstrap servers for Kafka clients.

> `_cluster-name_-kafka-external-bootstrap`:: Bootstrap service for clients connecting from outside the Kubernetes cluster. This resource will be created only when external listener is enabled.

Reviewer: ```suggestion
`_cluster-name_-kafka-external-bootstrap`:: Bootstrap service for clients connecting from outside the Kubernetes cluster. This resource is created only when external listener is enabled.
```

> `_cluster-name_-kafka-_pod-id_`:: Service used to route traffic from outside the Kubernetes cluster to individual pods. This resource will be created only when external listener is enabled.

Reviewer: ```suggestion
`_cluster-name_-kafka-_pod-id_`:: Service used to route traffic from outside the Kubernetes cluster to individual pods. This resource is created only when an external listener is enabled.
```

> `_cluster-name_-kafka-external-bootstrap`:: Bootstrap route for clients connecting from outside the Kubernetes cluster. This resource will be created only when external listener is enabled and set to type `route`.

Reviewer: ```suggestion
`_cluster-name_-kafka-external-bootstrap`:: Bootstrap route for clients connecting from outside the Kubernetes cluster. This resource is created only when an external listener is enabled and set to type `route`.
```

> `_cluster-name_-kafka-_pod-id_`:: Route for traffic from outside the Kubernetes cluster to individual pods. This resource will be created only when external listener is enabled and set to type `route`.

Reviewer: ```suggestion
`_cluster-name_-kafka-_pod-id_`:: Route for traffic from outside the Kubernetes cluster to individual pods. This resource is created only when an external listener is enabled and set to type `route`.
```

.Cruise Control

> These resources are only created if Cruise Control was deployed using the Cluster Operator.

Reviewer 2: not sure about "only created only"

Author: Adverb repetition. I think.
